app/renderer/components/settings.js

```diff
@@ -54,18 +67,34 @@ class SettingCheckbox extends ImmutableComponent {
   render () {
     const props = {
       style: this.props.style,
-      className: 'settingItem'
+      className: cx({
+        settingItem: true,
+        [this.props.className]: !!this.props.className
+      })
     }
+
     if (this.props.id) {
       props.id = this.props.id
     }
+
+    const labelClass = cx({
+      [css(settingCheckboxStyles.label)]: this.props.small,
+      [this.props.labelClassName]: !!this.props.labelClassName
+
+    })
+
     return <div {...props}>
       <SwitchControl id={this.props.prefKey}
         small={this.props.small}
         disabled={this.props.disabled}
         onClick={this.onClick}
-        checkedOn={this.props.checked !== undefined ? this.props.checked : getSetting(this.props.prefKey, this.props.settings)} />
-      <label className={css(this.props.small && settingCheckboxStyles.label)} data-l10n-id={this.props.dataL10nId} htmlFor={this.props.prefKey} />
+        checkedOn={this.props.checked !== undefined ? this.props.checked : getSetting(this.props.prefKey, this.props.settings)}
```

Comment:
I meant to `typeof this.props.checked !== 'undefined'`, but doesn't matter result is the same, let's keep as-is, sorry